tests/testthat/test-raster.R

```diff
@@ -1,5 +1,33 @@
 context("Raster conversion tests")
 
+skip_if_not_installed("terra")
```

Comment:
I'd suggest to put skip statements into the test case itself, otherwise the entire test file is skipped if terra is missing. In theory someone could have an old raster version installed, but not terra.

Reply:
I think that is ok.....the terra dependency in raster is already around for a while